tests/test_amplitude.py

```diff
@@ -61,6 +64,38 @@ def test_send_events(mocker):
         'json': {
             'events': events,
             'api_key': settings.API_KEY,
+            'options': {
+                'min_id_length': amplitude.min_id_length
+            }
+        }
+    }
+    request.assert_called_once_with(**kwargs)
+
+
+def test_send_events_with_min_id_length(mocker):
+    amplitude = Amplitude(min_id_length=100)
```

Comment:
As we test `min_id_length` works in the `test_init_pass_args()` test and `AMPLITUDE_MIN_ID_LENGTH` is going to be removed from the `test/settings.py`, could we run this by setting the `AMPLITUDE_MIN_ID_LENGTH` in the test?

```python
settings.AMPLITUDE_MIN_ID_LENGTH = 100
from amplitude import settings as appsettings
reload(appsettings)
amplitude = Amplitude()
```